tests/integration/test_benchmark.py

```diff
@@ -49,6 +49,21 @@ def test_benchmark_single_table_basic_synthsizers():
     ] == quality_scores.index.tolist()
 
 
+def test_benchmark_single_table_realtabformer_no_metrics():
+    """Test it without metrics."""
+    # Run
+    output = sdgym.benchmark_single_table(
+        synthesizers=['RealTabFormerSynthesizer'],
+        sdv_datasets=['student_placements'],
+        sdmetrics=[],
+    )
+
+    # Assert
+    assert set(output['Synthesizer']) == {'RealTabFormerSynthesizer'}
+    assert 'Train_Time' in output
+    assert 'Sample_Time' in output
```

Comment:
I would expect to validate that `output` contains actual values and there are no errors:
<img width="1136" alt="image" src="https://github.com/user-attachments/assets/0d32a97c-963a-4ed9-bb5b-22403d1bef00">